test/test_helper.rb

```diff
@@ -1,10 +1,17 @@
 require 'minitest/autorun'
 require 'semian'
 require 'semian/mysql2'
+require 'semian/redis'
 require 'toxiproxy'
 require 'timecop'
 require 'tempfile'
 require 'fileutils'
 
+require 'helpers/background_helper'
+
 Semian.logger = Logger.new(nil)
-Toxiproxy.populate(File.expand_path('../fixtures/toxiproxy.json', __FILE__))
+Toxiproxy.populate(File.expand_path('../helpers/toxiproxy.json', __FILE__))
```

Comment:
Off-side comment: I think `Toxiproxy.populate` should take a Hash, I may have gotten a little aggro in forcing people to store it in a JSON file. I still think that should be done in most use cases, but what do you think about making it a Hash? Would make the introduction simpler too.